src/App.css

```diff
@@ -0,0 +1,24 @@
+body {
```

Comment:
- [ ] [OPTIONAL] I guess you forgot to add the MIT license file. Please include it